phantombot/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  server:
+    image: ghcr.io/phantombot/phantombot:3.14.1.0@sha256:dbec9818e40f967ac5aee3abcac5a1857481cbbe6b35400f9a2fa8f1dc638df0
+    user: 0:900
+    restart: on-failure
+    volumes:
+      - ${APP_DATA_DIR}/data:/opt/PhantomBot_data
+    ports:
+      - "${APP_PHANTOM_SERVER_PORT}:${APP_PHANTOM_SERVER_PORT}"
+    environment:
+      PHANTOMBOT_USEHTTPS: "true"
```

Comment:
I haven't tested this submission yet, but I'd imagine this forces user to use https, so they'd click the app and go to https://umbrel.local:25000, is that right? If so they'll be met with the big scary insecure warning that we shouldn't be teaching the average umbrel user to just click through.

We can set this to false, so the user is accessing over http on their local network.